src/guides/v2.4/graphql/queries/pickup-locations.md

```diff
@@ -0,0 +1,203 @@
+
+The query returns `PickupLocations` object which contains information about pagination and list of `PickupLocation` items.
+
+## Syntax
+
+```graphql
+pickupLocations (area: AreaInput filters: PickupLocationFilterInput sort: PickupLocationSortInput pageSize: Int currentPage: Int): PickupLocations
+```
+
+## Example usage
+
+The following query returns the closest pickup location to `Austin` with limitation in 1500 KM.
+Because `pageSize` is set to 1 and distance by `distance` is applied, result will return the most closest pickup location.
+
+**Request:**
+
+```graphql
+{
+  pickupLocations(
+    area:{
+      radius: 1500
+      search_term: "Austin:US"
+    }
+    pageSize: 1
+    currentPage: 1
+    sort: {distance: ASC}
+  ) {
+    items {
+      pickup_location_code
+      name
+      email
+      fax
+      description
+      latitude
+      longitude
+      country_id
+      region_id
+      region
+      city
+      street
+      postcode
+      phone
+    },
+    total_count
+    page_info {
+      page_size
+      current_page
+      total_pages
+    }
+  }
+}
+```
+
+**Response:**
+
+```json
+{
+  "data": {
+    "pickupLocations": {
+      "items": [
+        {
+          "pickup_location_code": "txspeqs",
+          "name": "Sport Equipment Store",
+          "email": "sales@company.com",
+          "fax": null,
+          "description": "Sport Equipment Store description",
+          "latitude": 29.7543,
+          "longitude": -95.3609,
+          "country_id": "US",
+          "region_id": 57,
+          "region": "Texas",
+          "city": "Houston",
+          "street": "4631 Airport Blvd #125",
+          "postcode": "77010",
+          "phone": "(555) 555-5555"
+        }
+      ],
+      "total_count": 1,
+      "page_info": {
+        "page_size": 1,
+        "current_page": 1,
+        "total_pages": 1
+      }
+    }
+  }
+}
+```
+
+## Input attributes
+
+All attributes are optional. If no filters are provided, the query will return a list of pickup locations, assigned to the Sales Channel which is used by the store.
+
+Attribute | Data type | Description
+--- | --- | ---
+`area` | [AreaInput](#AreaInput) | Perform search by location using radius and search term.
+`filters` | [PickupLocationFilterInput](#PickupLocationFilterInput) | Apply filters by attributes.
+`sort` | [PickupLocationSortInput](#PickupLocationSortInput) | Specifies which attribute to sort on, and whether to return the results in ascending or descending order.
+`pageSize` | Int | The maximum number of pickup locations to return at once. The attribute is optional. The default value is 20.
+`currentPage` | Int | Specifies which page of results to return. The default value is 1.
+
+### AreaInput object {#AreaInput}
+
+Use the `AreaInput` to apply filtration by distance. All attributes are required.
+
+Attribute | Data type | Description
+--- | --- | ---
+`radius` | Int | The radius for the search in KM.
```

Comment:
If the attribute is mandatory, please, highlight it in the "Data type" column
```suggestion
`radius` | Int! | The radius for the search in KM.
```

It applies for all other required attributes